py-kms/pykms_Server.py

```diff
@@ -45,6 +45,7 @@ class KeyServer(socketserver.ThreadingMixIn, socketserver.TCPServer):
         allow_reuse_address = True
 
         def __init__(self, server_address, RequestHandlerClass):
+                self.address_family = socket.AF_INET6 # This call make sure the server creates an IPv6 socket and NOT an IPv4 by default
```

Comment:
Well that was a simple change! Nice job!

Reply:
Yeah, I didn't expected that either! I just had to read the library source code and understand at which point the socket was created :grin: